spec/requests/order_items_spec.rb

```diff
@@ -1,15 +1,38 @@
 describe "OrderItemsRequests", :type => :request do
   around do |example|
     bypass_rbac do
-      example.call
+      with_modified_env(:TOPOLOGICAL_INVENTORY_URL => "http://localhost") do
+        example.call
+      end
     end
   end
 
+  let(:service_plan_response) { TopologicalInventoryApiClient::ServicePlansCollection.new(:data => [plan]) }
+  let(:service_offering_response) do
+    TopologicalInventoryApiClient::ServiceOffering.new(:extra => {"survey_enabled" => true})
+  end
+  let(:plan) do
+    TopologicalInventoryApiClient::ServicePlan.new(
+      :name               => "Plan A",
+      :id                 => "1",
+      :description        => "Plan A",
+      :create_json_schema => {}
+    )
+  end
+
+  before do
+    stub_request(:get, "http://localhost/api/topological-inventory/v1.0/service_offerings/#{portfolio_item.service_offering_ref}")
+      .to_return(:status => 200, :body => service_offering_response.to_json, :headers => default_headers)
+    stub_request(:get, "http://localhost/api/topological-inventory/v1.0/service_offerings/#{portfolio_item.service_offering_ref}/service_plans")
+      .to_return(:status => 200, :body => service_plan_response.to_json, :headers => default_headers)
+  end
+
```

Comment:
All of this setup can probably just be condensed into stubbing `Catalog::ServicePlans` instead of making the request to topology, since you're already testing that within the spec for `Catalog::ServicePlans`.

Reply:
Stubbed them out.